### PR TITLE
build/ops: add jq runtime dependency to spec file

### PR DESCRIPTION
**Pull request by smithfarm**
Fixes: https://github.com/SUSE/DeepSea/issues/1218
Signed-off-by: Nathan Cutler <ncutler@suse.com>

### CODE REVIEW

**Comment by jan--f** [comment]
Huh I guess the jq install in packages.common is not enough?

**Comment by smithfarm** [comment]
RPM is a pretty good mechanism for installing dependencies of code (Salt Runners in this case) that runs on the machine where an RPM (deepsea in this case) is installed.

I understand that Stage 0 is supposed to install DeepSea's runtime dependencies on the minions that need them, and that's the only option for installing dependencies on the non-master minions, but for the master minion dependencies RPM has a big advantage in that RPM runs before any DeepSea code gets run.

In other words, given two mechanisms for installing dependencies - one that uses an external tool which runs before the dependent code runs, and another which relies on the dependent code itself to install the dependency, IMO the former will always be preferable.

**Comment by smithfarm** [comment]
backport is https://github.com/SUSE/DeepSea/pull/1225
